discourse_migration/discourse-categories.md

```diff
@@ -0,0 +1,75 @@
+# Proposed Discourse Categories
+
+* Announcements
+
+* Releases
+
+* Clang Compiler & Tools
+  - Using Clang
+  - Building Clang
+  - clangd
+
+* Subprojects
+  - lld
+  - Flang
+  - lldb
+  - Static Analyzer
+
+* Code Generation
+  - AArch64
+  - AMDGPU
+  - ARC
+  - ARM
+  - AVR
+  - BPF
+  - CSKY
+  - Hexagon
+  - Lanai
+  - M68k
+  - Mips
+  - MSP430
+  - NVPTX
+  - PowerPC
+  - RISCV
+  - Sparc
+  - SystemZ
+  - VE
+  - WebAssembly
+  - X86
+  - XCore
+
+* IR & Optimizations
+  - Loop Optimizations
+
+* Community
+  - Women in Compilers and Tools
+  - Job Postings
+  - US Developer Meeting
+  - EuroLLVM
+  - GSoC
+  - Community.o
+  - Foundation
+
+* Project Infrastructure
+  - Website
+  - Documentation
+  - GitHub
+  - Code Review
+  - Discord
+  - Mailing Lists
+  - IRC
+  - Infrastructure Working Group
+  - Policies
+
+* Runtimes
+  - C
+  - C++
+  - OpenCL
+  - OpenMP
+  - Sanitizers
+
+* Incubator
+  - CIRCT
+  - mlir-npcomp
+
+* LLVM Dev List Archives
```

Comment:
top level items get a lot of real estate on the front page, and this is only for historical reference / searching. Can this be "Community / Archived llvm-dev List" or something like that?

Reply:
I'm not loving this being under Community. I would rather it be under Project Infra if we aren't going to make it top-level. Or stick it in IR or something. Top level seems more appropriate though given it fits nothing of the top level categories exactly.

Reply:
"Project Infra" or "IR" would also be fine to me.   Top Level doesn't make sense because it will get a LOT of space on the [front page](https://llvm.discourse.group/) and be literally useless (in terms of not something anyone would click on).